source/extensions/filters/http/istio_stats/istio_stats.cc

```diff
@@ -290,6 +296,13 @@ struct Context : public Singleton::Instance {
   const Stats::StatName cluster_name_;
   const Stats::StatName app_name_;
   const Stats::StatName app_version_;
+
+  // istio_build metric.
+  const Stats::StatName istio_build_;
```

Comment:
Can you add a comment in the code to explain 1) what these metrics are and 2) example values of them?

Reply:
Added a link to the sample.

Reply:
testdata/metric/istio_build.yaml contains the following example values for component_ and tag_. Can you add example values for proxy_ and istio_version_?
  - name: component
    value: proxy
  - name: tag
    value: 1.5-dev

Reply:
proxy and istio_version are values for component and tag.

Reply:
Can you add example values for proxy and istio_version too?

Reply:
This might be confusing but "proxy" is the value :)
IstioVersion is just whatever ISTIO_VERSION label, will note.

Reply:
Do you mean "Stats::StatName proxy_" always have the string "proxy" as its value?
If not, can you provide an example value for "Stats::StatName proxy_"?

Reply:
Yes, proxy_ has value "proxy".

Reply:
Thanks! Can you add this information as a comment in the code?

Reply:
L190
```
 proxy_(pool_.add("proxy")),
 tag_(pool_.add("tag")),
```

Reply:
Right, this is just a symbol table, all variables are statically defined.